src/js/components/uploadFabsFile/UploadFabsFileValidation.jsx

```diff
@@ -364,36 +406,40 @@ export class UploadFabsFileValidation extends React.Component {
                             <div className="row">
                                 <div className="col-xs-8 button-text-container text-right">
                                     <Icons.CheckCircle />
-                                    File Published: {this.state.fabs_meta.valid_rows}&nbsp;
-                                    row(s) of data (excluding header) published at &nbsp;
+                                        File Published: {this.state.fabs_meta.valid_rows}&nbsp; row(s) of data
+                                        (excluding header) published at &nbsp;
                                     {parsedDate[0]} on {parsedDate[1]}
```

Comment:
Do these still need to be on a different line here? (nitpicking at this point, I know)